File: JavaScript/1036.js
var input = require('fs').readFileSync('./dev/stdin/file.txt', 'utf8');
var lines = input.split('\n');
const pegarValores = (line) => line.split(" ").map(a => +a);
let [A,B,C] = pegarValores(lines.shift())

let result  = Math.sqrt((B*B)-(4*A*C))
if(A != 0 && result > 0 ){
    let r1 = ((-B+result)/(2*A))
    let r2 = ((-B-result)/(2*A))
    console.log(`R1 = ${r1.toFixed(5)}`)
    console.log(`R2 = ${r2.toFixed(5)}`)
}else{
    console.log("Impossivel calcular")
}

File: Python/1036.py
import math
lista = (input().split(" "))
a,b,c = lista
a = float(a)
b = float(b)
c = float(c)
try:
    r1 = (-b+(math.sqrt((b*b)-4*a*c)))/(2*a)
    r2 = (-b-(math.sqrt((b*b)-4*a*c)))/(2*a)
    print("R1 = %.5f"%r1)
    print("R2 = %.5f"%r2)
except:
    print("Impossivel calcular")

